storage/merkle_path.go

```diff
@@ -0,0 +1,73 @@
+package storage
```

Comment:
NodeID belongs to storage though. I thought of splitting it to return abstract coords and then converting them to NodeID in storage but not sure it's worth it?